README.md

```diff
@@ -150,7 +150,7 @@ For `nvim-treesitter` to support a specific feature for a specific language requ
 
 The following is a list of languages for which a parser can be installed through `:TSInstall`; a checked box means that `nvim-treesitter` also contains queries at least for the `highlight` module.
 
-Experimental parsers are parsers that are maintained, but not stable enough for
+Experimental parsers are parsers that have a maintainer but are not stable enough for
 daily use yet. They are excluded from automatic installation when
 `ensure_installed` is set to `"maintained"`.
```

Comment:
here is still `maintained`

Reply:
Yes, on purpose, as the option is still there and works.

When the deprecation hits the fan, I will remove all mention of "maintained" from the repo with a hot `grep`.